### ISSUE TITLE
Telegram chat link is invalid

### ISSUE THREAD

**Issue by avdeev** [NNRUG/it52-rails#78]
https://web.telegram.org/#/im?tgaddr=tg:%2F%2Fjoin%3Finvite%3DAfMU4ABHPrqm36kOMmuB2Q

The invite link is invalid

```
Method: messages.checkChatInvite
Url: N/A
Result: {"_":"rpc_error","error_code":400,"error_message":"INVITE_HASH_EXPIRED"}
Stack: Error
    at u (https://web.telegram.org/js/app.js:29:4512)
    at https://web.telegram.org/js/app.js:29:5813
    at l (https://web.telegram.org/js/app.js:13:12014)
    at https://web.telegram.org/js/app.js:13:12186
    at m.$eval (https://web.telegram.org/js/app.js:13:19592)
    at m.$digest (https://web.telegram.org/js/app.js:13:18023)
    at https://web.telegram.org/js/app.js:13:19679
    at a (https://web.telegram.org/js/app.js:12:3095)
    at https://web.telegram.org/js/app.js:12:4549
    at n (https://web.telegram.org/js/app.js:3:16924)
```

**Comment by r3nya** [comment]
Hi @avdeev, thanks for your issue. We'll fix it soon.

btw, you can use -> https://telegram.me/nnrug

**Comment by top4ek** [comment]
Fixed 7a5734604b8e53712ba5cc3f0c1667fc3ee53638

**Comment by r3nya** [comment]
@top4ek :+1:
